### ISSUE TITLE
Trouble saving profile

### ISSUE THREAD

**Issue by simiusamoris** [neagle/gokibitz#208]
I apologize if this is just user error, but I'm having trouble saving my profile on the website. When I click edit, I am presented with the option to update my profile details. However, I am not able to save. The button is remains disabled, and if I navigate away from the page and navigate back nothing is saved.

**Comment by apetresc** [comment]
Hm, interesting. I just tested it out and had no issues. Do you know how to use your browser's Web Inspector? Does it show anything suspicious in there when you hit "Edit"?

**Comment by neagle** [comment]
Sorry you're having trouble! I just checked and had the same results as @apetresc. I've attached a video of my attempt to replicate the issue:

https://user-images.githubusercontent.com/176993/115890492-fcad6f00-a422-11eb-9eab-bf2ce3e1cfcc.mov

Could you try what @apetresc suggests: opening your browser's console and seeing if you get any errors when clicking edit or when trying to click save? And could you post what browser & version you're using?

**Comment by simiusamoris** [comment]
That's very interesting. I'm definitely not having the success you are in updating my profile. I do not know whether it is relevant, but I am a new user (signed up yesterday), so perhaps the issue is related to updating values in a new profile versus one that is existing?

I've attached a screenshot of my profile page with developer tools opened. I don't see anything fishy in the console, but when I inspect the "save" button, I can see that it is disabled. I'm guessing that's the root of my problem.

![ProfileScreenshot](https://user-images.githubusercontent.com/32541815/115929618-1ae09280-a456-11eb-97b7-c8d1d89daf08.png)

**Comment by simiusamoris** [comment]
I am using the following version of Google Chrome: Version 90.0.4430.85 (Official Build) (64-bit).

**Comment by simiusamoris** [comment]
I just tried on Microsoft Edge Version 90.0.818.46 (Official build) (64-bit). I had the same result.

**Comment by neagle** [comment]
Oof, I figured it out, but that is a _bad_ user experience. I apologize. The reason, I'm pretty sure, is that you have the "K" in your rank is capitalized, which the validator won't accept. If you change it to "k", I think it should work.

Development on GoKibitz is effectively pinned right now, unfortunately, due to how long it's been since I've upgraded a whole bunch of the tools necessary to build it. I should definitely improve the messaging so that users can't experience a totally mysterious issue like this, but I can't promise any timeline on that.
